test/integration/targets/eos_eapi/tests/cli/off.yaml

```diff
@@ -10,6 +10,7 @@
   register: eos_eapi_output
   connection: local
 
+- pause: seconds=20
```

Comment:
Could you please add a note to say why this is needed. 

I wonder if we night need to document this.